### PR TITLE
Create language_pu-in.properties

### PR DESCRIPTION
**Pull request by anupeshverma**

### CODE REVIEW

**Comment by anupeshverma** [comment]
Please accept the Pr and label as hacktoberfest

**Comment by AshishSharma1203** [comment]
@xEdziu  is this translation completed or can i work upon this ??? Because the PR hasn't been merged yet.

**Comment by xEdziu** [comment]
@AshishSharma1203 hi, I've quite busy recently, I will work on this repo today

File: src/main/resources/language_pu-in.properties [hunk 1]
```diff
@@ -0,0 +1,113 @@
+add=ਸ਼ਾਮਲ ਕਰੋ
+adding.new.password=Adding new password
```
Comment (xEdziu) [inline src/main/resources/language_pu-in.properties:2]: Not translated line

File: src/main/resources/language_pu-in.properties [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+appName=ਮੇਰਾ ਪਾਸਵਰਡ ਰੱਖੋ
```
Comment (xEdziu) [inline src/main/resources/language_pu-in.properties:3]: Keep ```appName``` key as given in Guide for Translators in wiki section :)

**Review by xEdziu — CHANGES_REQUESTED** [review]
Do not seperate add spaces (blanks) between key and value

**Comment by AshishSharma1203** [comment]
> @AshishSharma1203 hi, I've been quite busy recently, I will work on this repo today

It's okkayy sir. 
@anupeshverma  has already worked upon this. So , its better he will make the required changes (if soo) . I can work upon another issue.

**Review by xEdziu — APPROVED** [review]
Now everything is okay!

**Comment by xEdziu** [comment]
I have one question, what language is that? @anupeshverma

**Comment by anupeshverma** [comment]
Its Punjabi language spoken in Punjab state of India

**Pull request merged**